src/helpers/proofs/fetchCentralizedProof.ts

```diff
@@ -24,15 +24,7 @@ export default async function (
 }
 
 async function sendRequest(id: string, proverAddress: string) {
-  const { data } = await axios.get<RequestJobResult>(
-    `${proverAddress}/prove/${id}`,
-    {
-      headers: {
-        'User-Agent': 'SealHub',
-      },
-    }
-  )
-  return data
+  return await api.get(`${proverAddress}/prove/${id}`).json<RequestJobResult>()
```

Comment:
Just return promise

Reply:
There is no need to return Promise because `ky.METHOD().json()` return the data